.github/workflows/agent-build.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         arch: ['x64', 'x86']
         exclude:
         # Excludes PHP 5.x, 7.x on linux 32-bit
+        # Excludes PHP 8.x as only 64bit is supported for these versions
```

Comment:
Great start.  Can you please exclude `8.0-zts` and `8.1-zts` as well?

Reply:
Yes I'll fix that.